ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/replacehook.js

```diff
@@ -27,6 +27,7 @@
     const typeMap = {
         'button': fieldTypes.NON_INPUT,
         'checkbox-group': fieldTypes.SELECT,
+        'checkbox': fieldTypes.NON_INPUT,
```

Comment:
Why mapped to NON_INPUT? Are we suggesting Checkbox/Switch - can we replaced with Button?
cc: @sakshi-arora1

Reply:
I think there should be separate type for checkbox and switch so that only these can be interreplaced

Reply:
Not clear - why a separate type is needed. A customer can have 2 custom components extended from same type - and they should be replaceable among each other